RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -34,15 +34,21 @@
     overrideTrkQuals                         = 'detachedQuadStepSelector:detachedQuadStepTrk'
 ))
 from Configuration.ProcessModifiers.trackingIters01_cff import trackingIters01
-trackingIters01.toModify(convClusters,
-                         trajectories          = "highPtTripletStepTracks",
-                         oldClusterRemovalInfo = "highPtTripletStepClusters",
-                         overrideTrkQuals      = "highPtTripletStepSelector:highPtTripletStep"
-)
+(trackingIters01 & ~trackingPhase2PU140).toModify(convClusters,
```

Comment:
```suggestion
(trackingIters01 & trackingPhase1).toModify(convClusters,
```
perhaps may be more explicit (this is excluded from Phase2 and derived eras).
Clearly, this mix can be controlled on the workflow definition side; so the suggestion is somewhat minor